docs/website/contents/for-developers/utxo-hd/utxo-hd.md

```diff
@@ -0,0 +1,436 @@
+# UTXO-HD
+
+This document aims to provide an comprehensive guide on UTXO-HD, why we are
+implementing this feature, what brings to Cardano and what it implies.
+
+## Why does Cardano need UTXO-HD
+
+Cardano is built following the UTXO model. This means that the Ledger state
+contains a map from _transaction inputs_ to _transaction outputs_. A transaction
+might consume some of those entries and produce new ones. Each entry is owned by
+an account which is the one that can spend it.
+
+The UTXO set is an always growing data structure. Currently the `cardano-node`
+runs in about 16GBs of RAM but this amount will keep growing as more traffic
+takes place on the network. This is bad for decentralization and sustainability of
+the network as eventually only powerful machines would be able to participate on
+it.
+
+To improve decentralization, a decision was made to move this data to persistent
+storage which, albeit slower, is much cheaper than RAM. The Consensus layer is
+reworked so that the flow of data now allows for UTXO entries to come from some
+backend storage, which might be on disk or in memory trading memory for speed.
+
+The UTXO-HD feature provides two backends which can be chosen in cardano-node's
+configuration file:
```

Comment:
I think that we preferably shouldn't be talking about `cardano-node`'s configuration in file in consensus documents. Instead, I think we should just describe what the options are and that they can be set using arguments to the ChainDB/LedgerDB. The node can then have its own instructions/description for the config file fields